locale/en-us/dialog/notify_update_failure.dialog

```diff
@@ -0,0 +1 @@
+Something went wrong during the update. I am still running version {{version}}.
```

Comment:
This could be confusing for non-technical users. Maybe offer them the intent to submit a support ticket instead of just saying "something went wrong?"

Reply:
In theory this will only happen for pre-release versions but I agree the dialog should be actionable